javascript/ql/src/Vue/ArrowMethodOnVueInstance.ql

```diff
@@ -0,0 +1,19 @@
+/**
+ * @name Arrow method on Vue instance
+ * @description A arrow method on a Vue instance will not have its `this` variable bound to the Vue instance.
```

Comment:
Why are you using the future simple here + "a" should be "an"
I would suggest 
An arrow method on a Vue instance doesn't have its `this` variable bound to the Vue instance.